### ISSUE TITLE
[Feat]: Inconsistent Login/Logout Paths for OAuth Provider in Lowcoder Project

### ISSUE THREAD

**Issue by meong1234** [lowcoder-org/lowcoder#581]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When the OAuth provider is enabled, users are required to log in at the path "/org/:orgId/auth/login". However, upon logging out, users are redirected to a different path, specifically "/user/auth/login". This inconsistency in login and logout paths can lead to confusion and a less streamlined user experience.

### Expected Behavior

The ideal solution would be to have a unified login and logout path, ensuring consistency and ease of use. When an organization's user logs out, they should be redirected back to the organization-specific login page ("/org/:orgId/auth/login"), just like how they initially logged in. This would maintain a consistent user flow and reduce potential confusion.

### Steps to reproduce

Steps to Reproduce:

1. Enable OAuth provider in the Lowcoder project.
2. Navigate to the login page, and login
3. logout

**Comment by FalkWolsky** [comment]
You are right. This is not perfectly solved yet. Thank you very much for pointing it.
We see it as an enhancement and will solve it soon!
